controllers/scc.go

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
- please add this as one of resource unit,  just reminding

Reply:
done